kubernetes/structure_cron_job.go

```diff
@@ -5,7 +5,7 @@ import (
 	"k8s.io/api/batch/v1beta1"
 )
 
-func flattenCronJobSpec(in v1beta1.CronJobSpec, d *schema.ResourceData) ([]interface{}, error) {
+func flattenCronJobSpec(in v1beta1.CronJobSpec, d *schema.ResourceData, providerMetadata interface{}) ([]interface{}, error) {
```

Comment:
In most functions, we do have a `meta` argument that represents provider metadata data, but in some cases, this name is used to store Kubernetes object metadata. In such cases, I gave a new name to the provider metadata argument.
Shall I leave it that or stick with `meta` for the provider meta and rename Kubernetes object meta to something like `objectMeta` to avoid confusion?

Reply:
I think stick with `meta` for the moment just because it's so pervasive.